presto-main/src/main/java/com/facebook/presto/operator/aggregation/BooleanMinAggregation.java

```diff
@@ -21,7 +21,7 @@
 import static com.facebook.presto.operator.aggregation.state.TriStateBooleanState.NULL_VALUE;
 import static com.facebook.presto.operator.aggregation.state.TriStateBooleanState.TRUE_VALUE;
 
-@AggregationFunction("min")
+@AggregationFunction(value = "min", alias = "every")
```

Comment:
Hmm... This relies on booleans being orderable and in a particular way (which I don't thing should actually be allowed). I'd prefer to have an explicit implementation of this function